sdk/python/tests/compiler/testdata/withitem_nested.py

```diff
@@ -30,28 +30,28 @@ def get_code(self, ):
 
 @dsl.pipeline(name='my-pipeline')
 def pipeline(my_pipe_param: int = 10):
-    loop_args = [1, 2]
+    loop_args = [{'a': 1, 'b': 2}, {'a': 10, 'b': 20}]
```

Comment:
right now the loops tekton has only can take string or string Array argument. We cannot take dictionary array as it required us to change the tekton parameter api.

Reply:
Okay, let me revert the KFP 1.4.0 changes in the affected test cases for now and create a separate issue to address those